ProcessRequest.py

```diff
@@ -117,6 +117,6 @@ def processRequests(self):
                 self.proccesRequestType(requestItem)
             except Exception as e:
                 logger.error('invalid request')
-            finally:
                 requestItem.invalidRequest()
+            finally:
                 self.responder.sendResponse(requestItem)
```

Comment:
lets ditch the finally clause and move self.responder.sendResponse(reqItem) to the exception clause